src/newcharacter.cpp

```diff
@@ -1553,6 +1558,17 @@ tab_direction set_profession( avatar &u, points_left &points,
                     }
                 }
             }
+            // Proficiencies
+            const std::string newline = "\n";
```

Comment:
Wouldn't it be simpler to just use "\n" at those few places instead?

Reply:
I think it's cleaner doing it this way.